src/fix/rgbfix.1

```diff
@@ -179,6 +183,43 @@ sans global checksum:
 .Pp
 .D1 $ rgbfix -cjsv -k A4 -l 0x33 -m 0x1B -p 0xFF -r 3 -t SURVIVALKIDAVKE \
 SurvivalKids.gbc
+.Sh TPP1
```

Comment:
How about a brief note about what TPP1 is, linking to https://github.com/TwitchPlaysPokemon/tpp1?